1-run-automation-test.md

```diff
@@ -0,0 +1,122 @@
+
+  ![alt text](./assets/username.png )
+
+- API key 
+  + Select *user icon* -> *Settings* (you might find it in the top right corner)
+
+  ![alt text](./assets/api_key.png )
+
+### 2.2 Get desired capabilities
+The desired capabilities need to be added to the automation test script to allow the test to be executed on Kobiton device.
+  + Click *Devices* in the top navigation bar.
+ 
+    ![alt text](./assets/devices.png )
+ 
+  + Hover over any device you want to test with and click on the Automation settings button. A dialog box will then appear on screen.
+ 
+    ![alt text](./assets/click_auto.png )
+ 
+  + On the left hand side, you can select your preferred language, App Type, etc.
+
+    ![alt text](./assets/automation.png ) 
+### 2.3 Config TeamCity project
+Kobiton has already provided sample scripts for automation testing, visit [here](https://github.com/kobiton/samples) for reference.
+
+Create a Github repository for your script and configure with TeamCity.
+
+Follow steps below to setup TeamCity for automation testing.
++ In your Teamcity project, on the left side of the page, click *Build Step* to set build steps.
++ On the right side of the page, for the *Runner type* category, choose *Command Line*.
++ Add below commands to `Custom script` part.
+
+```
+  npm install
+  npm run android-app-test
+```
+> Note:
+> + `npm install` : install missing dependencies
+> + `npm run android-app-test` : run the test script
+
+![alt text](./assets/cmd.png)
+
+**(Optional) Attaching environment variable to Teamcity**
++ In your project in TeamCity, on the left side of the page, click *Parameters*. Then click `Add new parameter` button. 
+
+![alt text](./assets/param.png)
+
++ Add your Kobiton username and API key.
+
+![alt text](./assets/param_name.png)
+
+![alt text](./assets/param_key.png)
+### 2.4 Config automation test script
+Replace values of `username`, `apiKey` and `desiredCaps` in the script with the ones collected above.
+
+Example:
+
+```
+const username = {kobiton_username}
+const apiKey = {kobiton_api_key}
+ const kobitonServerConfig = {
+  protocol: 'https',
+  host: 'api-test.kobiton.com',
+  auth: `${username}:${apiKey}`
+}
+ var desiredCaps = {
+  sessionName:        'Automation test session',
+  sessionDescription: '', 
+  deviceOrientation:  'portrait',  
+  noReset:            true,
+  fullReset:          false, 
+  captureScreenshots: true,
+  app:                {your_app_id}, 
+  udid:               {your_device_udid}
+}
+```
+## 3. Run automation test on Kobiton devices
++ Push your changes to GitHub. TeamCity will install the neccessary dependencies and then run the test on Kobiton. 
+
+![alt text](./assets/build_complete.png )
+
++ After the script is executed successfully, you can access https://portal.kobiton.com/sessions to examine test results.
+
+![alt text](./assets/session.png )
+
++ Clicking on the latest session, you can see its detailed information.
```

Comment:
`Clicking` -> `Click`
`latest session` doesn't sound too clear. Is it always the latest session?